saleor/static/dashboard-next/pages/views/PageDetails.tsx

```diff
@@ -33,102 +31,88 @@ export class PageUpdateForm extends React.Component<
     const { id } = this.props;
     return (
       <TypedPageDetailsQuery displayLoader variables={{ id }}>
-        {({ data: detailsResult, error, loading }) => {
-          if (error) {
-            return (
-              <ErrorMessageCard
-                message={i18n.t("Unable to find matching page.")}
-              />
-            );
-          }
-          return (
-            <TypedPageDeleteMutation>
-              {(
-                _,
-                { called, data: deleteResult, error, loading: deleteInProgress }
-              ) => {
-                if (called && !deleteInProgress) {
-                  if (error) {
-                    console.error(error);
-                    return;
-                  }
-                  if (
-                    deleteResult.pageDelete.errors &&
-                    deleteResult.pageDelete.errors.length
-                  ) {
-                    console.error(deleteResult.pageDelete);
-                    return;
-                  }
-                  // FIXME: component is loaded with previous state (meaning that deleted page will still be there until table reload)
-                  this.handleRemoveButtonClick();
-                  return <Redirect to={pageListUrl} />;
+        {({ data, loading }) => (
+          <TypedPageDeleteMutation>
+            {(
+              _,
+              { called, data: deleteResult, error, loading: deleteInProgress }
+            ) => {
+              if (called && !deleteInProgress) {
+                if (error) {
+                  console.error(error);
+                  return;
                 }
-                return (
-                  <TypedPageUpdateMutation>
-                    {(
-                      updatePage,
-                      {
-                        called,
-                        data: updateResult,
-                        error,
-                        loading: updateInProgress
-                      }
-                    ) => {
-                      if (
-                        called &&
-                        !updateInProgress &&
-                        updateResult &&
-                        updateResult.pageUpdate &&
-                        !updateResult.pageUpdate.errors.length
-                      ) {
-                        if (error) {
-                          console.error(error);
-                          return;
-                        }
-                        return <Redirect to={pageListUrl} />;
+                if (
+                  deleteResult.pageDelete.errors &&
+                  deleteResult.pageDelete.errors.length
+                ) {
+                  console.error(deleteResult.pageDelete);
```

Comment:
I think we can get rid of all `console.error` - we'll show a message that something failed and the developer would debug this in browser developer tools anyway.